src/main/java/com/minecolonies/coremod/colony/Structures.java

```diff
@@ -115,18 +115,30 @@ public static void init()
      * Loads all styles saved in ["/assets/minecolonies/schematics/"].
      * Puts these in {@link #md5Map}, with key being the fullname of the structure (schematics/stone/Builder1).
      */
+    // The same exception will be triggered in the 2nd catch with logging this time.
+    @SuppressWarnings("squid:S1166")
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png 'Severity: INFO') Suppressing the 'squid:S1166' warning is not allowed [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1309)